src/listen.js

```diff
@@ -45,6 +45,9 @@ module.exports = function(defaultFuncs, api, ctx) {
       case 'read_receipt':
         globalCallback(null, utils.formatReadReceipt(event));
         return true;
+      case 'read':
```

Comment:
Just in case you didn't know, you should be able to update this pull request with the new commits rather than creating a whole new one.